### PR TITLE
Switch to PHPicker

### PR DESCRIPTION
**Pull request by theamandawang**
### Description
Provides more intuitive functionality to select multiple images.

### Milestones
N/a

### Demo

https://user-images.githubusercontent.com/52474776/182198355-1f98b8ab-e88a-4639-b7f0-516f75be4c51.mp4

### CODE REVIEW

File: Enable/View Controllers/ComposeViewController.m [hunk 1]
```diff
@@ -313,4 +289,30 @@ - (void) setupTheme{
     [self.photosImageView setTintColor: [UIColor colorNamed: colorSet[@"Accent"]]];
 }
 
+- (void)picker:(nonnull PHPickerViewController *)picker didFinishPicking:(nonnull NSArray<PHPickerResult *> *)results {
+    int count = 0;
+    if(self.images.count >= 3){
```
Comment (metacaprio) [inline Enable/View Controllers/ComposeViewController.m]: why the >= 3?

File: Enable/View Controllers/ComposeViewController.m [hunk 2]
```diff
@@ -313,4 +289,30 @@ - (void) setupTheme{
...
+    }
+    for (PHPickerResult * res in results){
+        if([res.itemProvider canLoadObjectOfClass:[UIImage class]]){
+            [res.itemProvider loadObjectOfClass:[UIImage class] completionHandler:^(__kindof id<NSItemProviderReading>  _Nullable object, NSError * _Nullable error) {
+                if(object){
+                    dispatch_async(dispatch_get_main_queue(), ^{
+                        if(count == 0) {
+                            self.photosImageView.image = (UIImage*)object;
+                        }
+                        if(self.images.count < 3){
```
Comment (metacaprio) [inline Enable/View Controllers/ComposeViewController.m]: again, same thing

**Review by metacaprio — CHANGES_REQUESTED** [review]

File: Enable/View Controllers/ComposeViewController.m [hunk 3]
```diff
@@ -313,4 +289,29 @@ - (void) setupTheme{
...
+    [self.images removeAllObjects];
+    for (PHPickerResult * res in results){
+        if([res.itemProvider canLoadObjectOfClass:[UIImage class]]){
+            [res.itemProvider loadObjectOfClass:[UIImage class] completionHandler:^(__kindof id<NSItemProviderReading>  _Nullable object, NSError * _Nullable error) {
+                if(object){
+                    dispatch_async(dispatch_get_main_queue(), ^{
+                        if(count == 0) {
+                            self.photosImageView.image = (UIImage*)object;
+                        }
+                        if(self.images.count < kMaxNumberOfImages){
```
Comment (metacaprio) [inline Enable/View Controllers/ComposeViewController.m:304]: Ah got it, do we have some feedback if someone chooses more than max number?

**Review by metacaprio — APPROVED** [review]

File: Enable/View Controllers/ComposeViewController.m (hunk 3 above)
Comment (theamandawang) [reply Enable/View Controllers/ComposeViewController.m:304]: PHPicker makes it impossible for someone to choose more than the max; I mostly have this code here to safeguard

**Pull request merged**